RESTserver/src/main/java/com/softserve/if072/restservice/service/impl/ShoppingListServiceImpl.java

```diff
@@ -15,14 +16,23 @@
     private ShoppingListDAO shoppingListDAO;
 
     @Override
-    public List<ShoppingList> getByUserId(int user_id) {
-        return shoppingListDAO.getByUserID(user_id);
-
+    public List<ShoppingList> getByUserId(int user_id) throws DataSourceException {
+        List<ShoppingList> list = shoppingListDAO.getByUserID(user_id);
+        if (list != null && !list.isEmpty()){
```

Comment:
please check appache commons library and class with method CollectionUtils.isNotEmpty()